adapter.js

```diff
@@ -34,7 +34,7 @@ export function adapter(svcName, aws) {
   const receiveMessage = Async.fromPromise(aws.receiveMessage);
   const asyncFetch = Async.fromPromise(fetch);
   const deleteMessage = Async.fromPromise(aws.deleteMessage);
-  const listObjects = Async.fromPromise(aws.listObjects)
+  const listObjects = Async.fromPromise(aws.listObjects);
```

Comment:
Can't comment on the actual line, but ln.44 doesn't look to be defined. I think you need to import `processTasks`

Reply:
Good catch